### ISSUE TITLE
Not working when defining a custom edit_handler

### ISSUE THREAD

**Issue by pyMan** [infoportugal/wagtail-modeltranslation#85]
wagtail-modeltranslation 0.5.0a1

When you define a custom edit_handler ([look here](http://docs.wagtail.io/en/v1.6.2/advanced_topics/customisation/page_editing_interface.html?highlight=edit_handlers#customising-the-tabbed-interface)) in a model with any translatable field, things go wrong: you don't see any translatable field in the editor window, like if the model wasn't a translatable one.

The next sample works correctly only if the edit_handler definition is kept commented:

```
class TestPage(TranslationMixin, Page):
    body = StreamField(FullStreamBlock())

    content_panels = Page.content_panels + [
        FieldPanel('title', classname="full title"),
        StreamFieldPanel('body'),
    ]

    #It doesn't work if the next is uncommented!!!
#     edit_handler = TabbedInterface([
#         ObjectList(content_panels, heading='Content'),
#         ObjectList(Page.promote_panels, heading='Promote'),
#         ObjectList(Page.settings_panels, heading='Settings', classname="settings"),
#     ])

    search_fields = Page.search_fields + (
        index.SearchField('body'),
    )
```

translation.py

```
from wagtail_modeltranslation.translation import TranslationOptions
from wagtail_modeltranslation.decorators import register

from .models import TestPage

@register(TestPage)
class TestPageTR(TranslationOptions):
    fields = (
        'body',
    )
```

**Comment by pyMan** [comment]
This happens because WagtailTranslator (patch_wagtailadmin.py) patches the (only 3 standard) panels and set them as new attributes to the model. The pre existing 'edit_handler' still points to the old ones and those are used for display.
We need a nice way to rebuild the 'edit_handler' attribute too after the panel attribute sets. Please note that this is what exactly happens when 'edit_handler' is not custom-defined in the model.

**Comment by Alexandre27** [comment]
Fixed in version 0.6
